### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by tonglil**

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -7,39 +7,146 @@
...
+# Drone 1.0+ alternative
+---
+kind: pipeline
+# ...
+steps:
+  - name: deploy
+    image: nytimes/drone-gae
+    settings:
+      gae_credentials:
+        from_secret: GOOGLE_CREDENTIALS
```
Comment (oliver-nyt) [inline DOCS.md:42]: Thanks for adding this, didn't have a chance to get around to it in #62

File: main.go [hunk 2]
```diff
@@ -293,13 +293,13 @@ func configFromEnv(vargs *GAE, workspace *string) error {
 func validateVargs(vargs *GAE) error {
 
 	if vargs.Token == "" {
-		return fmt.Errorf("missing required param: token")
+		return fmt.Errorf("missing required credentials: GAE_CREDENTIALS or param token")
```
Comment (msuterski) [inline main.go:296]: Is the plugin looking for a `TOKEN` or `PLUGIN_TOKEN`?

Comment (tonglil) [reply main.go:296]: No, this plugin is looking for `GAE_CREDENTIALS`. I know, it's not the same. Maybe future improvement can look for multiple but the way it is written now it doesn't.

**Comment by tonglil** [comment]
We can tag a `0.1.0` release after this, then make changes to remove support for Drone 0.4 and release `1.0.0`, and in the future remove support for Drone 0.8 and release `2.0.0`.

File: main.go (hunk 2 above)
Comment (msuterski) [reply main.go:296]: I was referring more to the error message where it says `GAE_CREDENTIALS or param token`

**Pull request merged**